contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -200,13 +129,13 @@ <h2 id="title"> Contributions Catalog/Packager </h2>
             {% if cap_json|length > 0 %}
                 {% for capability in cap_json %}
                     <div class="column capability">
-                        <div class="content">
-                            <a class="card=link" href="{{ url_for('contribute.capability_details', contribution_id=capability['cont_id'], id=capability['id']) }}">
-                            <h4 class="capability_name">{{ capability|filter_nested_dict(["name"]) }}</h4>
-                            <p class="capability_description">{{ capability|filter_nested_dict(["description"]) }}</p>
-                            <p class="contributors">{{ capability|filter_nested_dict(["contributors"]) }}</p>
+                        <div class="card">
+                            <a class="card-link" href="{{ url_for('contribute.capability_details', contribution_id=capability['cont_id'], id=capability['id']) }}">
+                                <h3 class="capability_name">{{ capability|filter_nested_dict(["name"]) }}</h3>
```

Comment:
The capability name font weight is bold according to Zeplin and not 800 px. Please fix this. You could have a separate CSS class for the specific changes for the capability/talent name. The rest could be inherited from h3.

Reply:
Thanks.